package.json

```diff
@@ -45,35 +46,38 @@
     "rlp": "2.2.3",
     "solidity-coverage": "0.5.11",
     "solparse": "2.2.5",
-    "truffle": "5.0.19",
+    "truffle": "beta",
     "ts-node": "8.1.0",
-    "typescript": "3.4.5",
+    "typescript": "3.5.3",
     "wait-port": "0.2.2",
     "web3": "1.0.0-beta.37",
     "ts-generator": "0.0.8",
-    "typechain": "0.3.14"
+    "typechain": "0.3.14",
+    "fs-extra": "7.0.1"
   },
   "scripts": {
     "update": "git submodule update --init --recursive && npm ci",
     "compile": "truffle compile",
-    "compile:ts": "tsc --target es5",
-    "compile:all": "truffle compile --all && tsc --target es5",
-    "generate_test_proofs": "./proof_generation/main.sh",
+    "compile:ts": "tsc --resolveJsonModule",
+    "compile:all": "truffle compile --all && tsc --resolveJsonModule",
+    "generate:test_proofs": "./proof_generation/main.sh",
     "test": "npm run test:deployment_tool && npm run test:fuzzy_proof_generator && npm run test:integration && npm run test:unit && npm run build-package",
     "test:range": "./tools/test_range.sh",
     "test:deployment_tool": "mocha tools/deployment_tool/test",
     "test:fuzzy_proof_generator": "./tools/fuzzy_proof_generator_tool/test/run.sh",
     "test:integration": "cd test_integration && ./main.sh",
     "test:unit": "truffle test",
     "ganache": "sh tools/runGanacheCli.sh",
-    "build-package": "node tools/build_package.js",
-    "prepare": "npm run compile:all && npm run build-package",
+    "build:package": "node tools/build_package.js",
     "deploy:gateway": "npm run compile && node tools/blue_deployment/scripts/step1_origin_contracts.js",
     "lint": "eslint {test,test_integration,tools,proof_generation} -c .eslintrc.json --ext .js --ext .ts",
-    "generate:interacts": "ts-generator ts-generator.json",
-    "postinstall": "npm run compile:all && npm run generate:interacts"
+    "generate:interacts": "ts-generator ts-generator.json && node tools/contract_interact_generator.js",
+    "clean": "rm -r contract_build/contracts.json dist/* interacts/* build/*",
```

Comment:
This command will fail if contract/build/contracts.json or dist or interacts or build does not exist. Believe, this is not what we need. Hence, I would propose to do the following:

```bash
rm -r contract_build/contracts.json dist/* interacts/* build/* 2> /dev/null || true
```

Reply:
done 👍